Example/AztecUITests/AztecUITests.swift

```diff
@@ -0,0 +1,263 @@
+        richContentTextView.enterAndSelectText(text: "text")
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.unorderedlistButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<ul><li>text</li></ul>"
+        XCTAssertEqual(expected, text)
+    }
+
+    // Enable this test after ordered lists are fully implemented
+    func testSimpleOrderedListText() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+        richContentTextView.enterAndSelectText(text: "text")
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.orderedlistButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<ol><li>text</li></ol>"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testSimpleLinkedText() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+        richContentTextView.enterAndSelectText(text: "text")
+
+        // Copy link to be auto-filled in URL field
+        UIPasteboard.general.string = "https://wordpress.com/"
+        app.scrollViews.otherElements.buttons[elementStringIDs.linkButton].tap()
+        app.alerts.buttons.element(boundBy: 3).tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<a href=\"https://wordpress.com/\">text</a>"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testHorizontalRuler() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.horizontalrulerButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let tex: String = htmlContentTextView.value as! String
+        let expected = "<hr>"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testMoreTag() {
+        richContentTextView = app.textViews[elementStringIDs.richTextField]
+        richContentTextView.tap()
+
+        app.scrollViews.otherElements.buttons[elementStringIDs.moreButton].tap()
+        app.buttons[elementStringIDs.sourcecodeButton].tap()
+
+        htmlContentTextView = app.textViews[elementStringIDs.htmlTextField]
+        let text: String = htmlContentTextView.value as! String
+        let expected = "<!--more-->"
+        XCTAssertEqual(expected, text)
+    }
+
+    func testHeadingOneText() {
```

Comment:
and that it isn't applied to text prior to the headings. and that two different heading types can be next to each other. probably enough there that it's worth making a separate file of tests for headings.

This would be fine as a separate PR.